system-tests/project-fixtures/angular/src/app/mount.cy.ts

```diff
@@ -343,6 +345,35 @@ describe("angular mount", () => {
     cy.get('img').should('be.visible').and('have.prop', 'naturalWidth').should('be.greaterThan', 0)
   })
 
+  it('should not override transient service', () => {
+    cy.mount(TransientServicesComponent)
```

Comment:
I didn't know you could do this (rely on `@Injectable({ providedIn: 'root' })` to inject into the TestModule). Really cool!